ramls/schemas/user-import-request-preference.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request preference schema",
+  "properties": {
+    "id": {
+      "description": "Unique request preference ID",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
```

Comment:
uuid.schema from raml-util can be used in this place

Reply:
thanks, fixed